### PR TITLE
Make human activity recognition permission optional

### LINKED ISSUE #32: As a developer, I would like to be able to manually choose if I want to include the activity tracking permission in my app or not

### PR DESCRIPTION
**Pull request by agonper**
Fixes #32 

Previously anyone installing this plugin was forced to include the human activity recognition permissions in the app manifest, even if they only wanted to use the geolocation features. 

Following what was already done with the geolocation feature, with this PR activity recognition permissions will no longer be a must.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,26 +42,39 @@ android {
...
-  <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
+<!-- Required only when requesting background location access on Android 10 (API level 29) and higher. -->
+<uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
 ```
 
 More information can be found in the [Android docs here](https://developer.android.com/training/location/permissions).
 
 > Source: [https://github.com/NativeScript/nativescript-geolocation](https://github.com/NativeScript/nativescript-geolocation)
 
+#### Human activity detection
```
Comment (matey97) [inline README.md:66]: One more # to be at the same level as Geolocation section? 😉

**Review by matey97 — APPROVED** [review]
Nice idea to make the permission optional 👌.

File: README.md (hunk 1 above)
Comment (agonper) [reply README.md:66]: Ooops, true hehe

**Comment by agonper** [comment]
Thanks for the quick review @matey97!!

I'm going to merge this now, and I will release it as a new major version (due to the breaking change)

**Pull request merged**